src/data/nat/prime.lean

```diff
@@ -1196,13 +1196,11 @@ end int
 section
 open finset
 /-- Exactly `n / p` naturals in `[1, n]` are multiples of `p`. -/
-lemma card_multiples (n p : ℕ) : card {e ∈ range n | p ∣ e + 1} = n / p :=
```

Comment:
Oh, I hadn't noticed this was `finset.sep`. Maybe we  should be using `finset.sep` more often, perhaps even in preference to the `filter` spelling!